index.bs

```diff
@@ -409,13 +409,17 @@ However, an event whose only effect is to begin or update a fling or scroll
 gesture is not an excluding input.
 
 The user agent may delay the reporting of layout shifts after a
-<a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event
-until such time as it is known that the event does not begin a fling or scroll
-gesture.
-
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+<a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event or a
+<a href="https://www.w3.org/TR/uievents/#event-type-mousedown">mousedown</a> event until a
+<a href="https://www.w3.org/TR/pointerevents/#the-pointerup-event">pointerup</a> event or a
+<a href="https://www.w3.org/TR/uievents/#event-type-mouseup">mouseup</a> event are received, which turn into touch and mouse drag actions.
+These <a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a> events,
+<a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> events, 
+<a href="https://www.w3.org/TR/pointerevents/#the-pointerup-event">pointerup</a> event and
+<a href="https://www.w3.org/TR/uievents/#event-type-mouseup">mouseup</a> event are all excluding inputs.
+However, if a <a href="https://www.w3.org/TR/pointerevents/#the-pointercancel-event">pointercancel</a> event
```

Comment:
I think it will be clearer to make this into a list depending on which event is being dispatched by the user agent. Something like
* When down -> start buffering
* When pointercancel -> report shifts but not considering the inputs
* When up -> update the last input timestamp, and then report shifts

Reply:
ACK.